include/verilated_funcs.h

```diff
@@ -2339,6 +2339,60 @@ inline IData VL_VALUEPLUSARGS_INQ(int rbits, const std::string& ld, double& rdr)
 }
 extern IData VL_VALUEPLUSARGS_INN(int, const std::string& ld, std::string& rdr) VL_MT_SAFE;
 
+//======================================================================
+// bitHelper
+
+template <typename OutWordT, typename InWordT>
+void bitHelper(OutWordT* outWords, uint32_t outLsb, const InWordT* inWords, uint32_t inLsb,
+               uint32_t width) {
+    if (width == 0) return;
+    uint32_t outSizeb = sizeof(OutWordT) * 8;
+    uint32_t outIndexOff = outLsb / outSizeb;
+    uint32_t outBitOff = outLsb % outSizeb;
```

Comment:
```suggestion
    constexpr uint32_t outSizeb = sizeof(OutWordT) * 8;
    constexpr uint32_t outIndexOff = outLsb / outSizeb;
    constexpr uint32_t outBitOff = outLsb % outSizeb;
```

Or something else so / and % aren't slow operations.  Similar below.  Also add "const" to where can in below.

Reply:
Added `constexpr` and `const` qualifier as much as possible